doc/changelog.txt

```diff
@@ -32,6 +32,63 @@ BREAKING CHANGES & HOW TO GET BACK OLD BEHAVIOURS:
      return originalGetConfigInt(key, def) or 0
    end
 
+ ! hovercraft and ships brought out of water no longer forced to be upright.
+   To get back previous behaviour, put the `upright = true` tag in all unit defs
+   whose move def is of the hovercraft or ship type.
+
+ ! The return value from the UnitUnitCollision callin is now ignored
+   and there is only one event for each collision.
+   There is no way to get back the old behaviour for now,
+   but if someone needs it it could be arranged.
+
+ ! The GameProgress callin is now called every 5 seconds (from 10).
+   This value is not exposed anywhere so to get back the previous
+   behaviour you will have to find whichever code assumes the old value.
+
+ ! Buildings can be placed inside other buildings' open yardmap gaps.
+   Use the AllowCommand or AllowUnitCreation callins to prevent this.
```

Comment:
I am unsure which of those gives the exact previous behaviour (was the order dropped immediately or on constructor arrival at the site?) and a bit too low on mana to check and write a drop-in replacement, but it shouldn't be too hard.

Reply:
Stacking is enabled with certain new yardmap letters, without them the behavior should remain as it used to be.